apps/jetbrains.py

```diff
@@ -350,6 +350,7 @@ def phones_selection(m):
     "cut": [idea("action EditorCut")],
     "clear": [idea("action EditorBackSpace")],
     "comment": [idea("action CommentByLineComment")],
+    "paste": [idea("action EditorPaste")],
```

Comment:
paste with selection might be better called "replace"
I like having a paste for addition, but it might need some commands between the movement and the paste.

Reply:
possibly that would be ok with me. i primarily use the form "paste line x" to insert copied code on line x.

Reply:
maybe paste should be a movement verb as well?

Reply:
Paste as movement, replace as selection?